packages/react-devtools-extensions/src/background/dynamicallyInjectContentScripts.js

```diff
@@ -11,7 +11,7 @@ const contentScriptsToInject = IS_FIREFOX
         js: ['build/proxy.js'],
         matches: ['<all_urls>'],
         persistAcrossSessions: true,
-        runAt: 'document_idle',
+        runAt: 'document_end',
```

Comment:
just a typo fix, accidentally missed it in https://github.com/facebook/react/pull/27336, should not affect in any way

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/extensionTypes/RunAt